audbcards/core/templates/datacard_tables.j2

```diff
@@ -0,0 +1,11 @@
+
+Tables
+^^^^^^
+
+.. csv-table::
+   :header: ID,Type,Columns
+   :widths: 20, 10, 70
+
+{% for data in tables|zip(types)|zip(columns) %}
+    "{{ data[0][0] }}", "{{ data[0][1] }}",  "{{ data[1] }}"
```

Comment:
This fixes

> In content we get an extra space in the CSV file listing the tables between the Type and Columns entries, e.g. "files", "filewise",  "speaker"

from https://github.com/audeering/audbcards/pull/13#pullrequestreview-1648095651

```suggestion
    "{{ data[0][0] }}", "{{ data[0][1] }}", "{{ data[1] }}"
```

Reply:
Was added by https://github.com/audeering/audbcards/pull/23